test/core/fixtures/esnext/es6-templates/tagged-template-expression.js

```diff
@@ -24,3 +25,7 @@ function tagInterpolateLast(strings) {
   return 'whatever';
 }
 assert.equal(tagInterpolateLast `a${ 42 }`, 'whatever');
+
+// ENsure that we don't have conflicts between distinct templates with similar strings.
+assert.equal(String.raw `${ 42 }b`, '42b');
+assert.equal(String.raw `b${ 42 }`, 'b42');
```

Comment:
This is the wrong place for these tests. esnext contains regressions tests from the [esnext merger](http://babeljs.io/blog/2015/01/12/6to5-esnext/).